doc/faq.rst

```diff
@@ -34,11 +34,15 @@ Copy one of these links, making sure to include the lengthy token, and navigate
 web browser on your local computer. You will be able to use EOS within an interactive Python environment.
 
 You can make this more persistent by configuring your SSH connection to ```REMOTE_HOST```. Please see the documentation
-of your local computer's secure shell programm.
+of your local computer's secure shell program.
 
 ... use EOS on Windows?
 =======================
 
-Presently we do not support using EOS on Windows. We recommend that you install EOS on a
-remote Linux machine (e.g., one of your department's workstations) and connect remotely
-as discussed :ref:`above <... use EOS remotely?>`.
+EOS can be used on Windows by making use of the Windows Subsystem for Linux (WSL)
+which is offered and developed by Microsoft themselves.
+The installation of WSL is described in detail `here <https://docs.microsoft.com/en-us/windows/wsl/install>`__.
+This provides you with a well optimized virtual machine.
```

Comment:
Overly specific adjective: ``well optimized``. Instead: ``This provides you with a Linux container in virtual machine.``, maybe?

Reply:
The point I intended to make with this phrasing, is that I feel like WSL is a far superior solution and comes with a way better user experience than a Virtual Machine using e.g. HyperV